src/System.Collections/tests/Generic/HashSet/HashSet_ConstructorTests.cs

```diff
@@ -281,5 +281,27 @@ public static void HashSetConstructor_IEn_IEq_Comparer_NotIEquatable()
         }
 
         #endregion
+
+        #region Constructor_Capacity
+
+        //Test 1: Check that EqualityComparer<T>.Default is used
+        [Fact]
+        public static void HashSetConstructor_Capacity()
+        {
+            HashSet<int> hashSet = new HashSet<int>(13);
+            int[] items = new int[0];
+            HashSetTestSupport<int> driver = new HashSetTestSupport<int>(hashSet, s_intGenerator, items);
+            driver.VerifyHashSetTests();
+        }
+        [Fact]
+        public static void HashSetConstructor_Capacity_Negative()
+        {
+            HashSet<int> hashSet = new HashSet<int>(13);
+            int[] items = new int[0];
+            HashSetTestSupport<int> driver = new HashSetTestSupport<int>(hashSet, s_intGenerator, items);
+            driver.VerifyHashSet_NegativeTests();
+        }
+
+        #endregion
     }
```

Comment:
More tests should be done once the tooling problems are resolved related to API changes and unit testing.